File: php/update_subuser.php
<?php

// Information obtained from authentication

$userId = "";
$token = "";
$subUserId = 0; // Retrieved from Get Subusers

$url = 'https://api.smartproxy.com/v1/users/' . $userId . '/sub-users/' . $subUserId;

$data = array(
  "password" => "",
  "traffic_limit" => 0 // Optional parameter
);

$payload = json_encode($data);

$ch = curl_init();

curl_setopt($ch, CURLOPT_URL, $url);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "PUT");

$headers = array();
$headers[] = "Accept: application/json";
$headers[] = "Content-Type: application/json";
$headers[] = "Authorization: Token $token";
curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
curl_setopt($ch, CURLOPT_POSTFIELDS, $payload);

$result = curl_exec($ch);
if (curl_errno($ch)) {
    echo "Error: " . curl_error($ch);
}
curl_close($ch);
echo $result;

File: csharp/update_subuser.cs
using System;
using System.IO;
using System.Net;
using System.Text;

namespace Smartproxy_API
{
    class update_subuser
    {
        static void Main(string[] args)
        {
            // Details from authentication
            string userId = "";
            string token = "";
            int sub_user_id = null; // Recieved from get_subusers.cs

            string uri = "https://api.smartproxy.com/v1/users/" + userId + "/sub-users/" + sub_user_id;

            // Change null accordingly to the parameter
            string data = "{\"traffic_limit\":null,\"password\":\"null\"}";

            byte[] dataBytes = Encoding.UTF8.GetBytes(data);

            HttpWebRequest request = (HttpWebRequest)WebRequest.Create(uri);
            request.AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
            request.PreAuthenticate = true;
            request.Headers.Add("Authorization", "Token " + token);
            request.ContentType = "application/json";
            request.ContentLength = dataBytes.Length;
            request.Method = "PUT";

            using (Stream requestBody = request.GetRequestStream())
            {
                requestBody.Write(dataBytes, 0, dataBytes.Length);
            }

            using (HttpWebResponse response = (HttpWebResponse)request.GetResponse())
            using (Stream stream = response.GetResponseStream())
            using (StreamReader reader = new StreamReader(stream))
            {
                Console.WriteLine(reader.ReadToEnd());
                Console.ReadKey();
            }
        }
    }
}
